config/fgcmBuildStars.py

```diff
@@ -8,20 +8,20 @@
 # The mapping from HSC short filter name to FGCM "band".  This mapping
 # ensures that stars observed in HSC-R + HSC-R2 can be matched, and
 # HSC-I + HSC-I2 can be matched.
-config.filterMap = {'g': 'g', 'r': 'r', 'r2': 'r', 'i': 'i', 'i2': 'i',
-                    'z': 'z', 'y': 'y',
-                    'N387': 'N387', 'N816': 'N816', 'N921': 'N921',
-                    'N1010': 'N1010'}
+config.physicalFilterMap = {'HSC-G': 'g', 'HSC-R': 'r', 'HSC-R2': 'r',
```

Comment:
It's still vaguely annoying that this mapping has to be duplicated since it's defined in filterDefinitions.py as a source of fundamental truth.

Reply:
After chatting with @parejkoj I think it might be possible to use the filter labels instead, and remove this entirely.